apps/sample-configs/renderManager.extended.portrait.json

```diff
@@ -23,6 +24,14 @@
             "bitsPerColor": 8
         },
 
+		"prediction": {
+				"enabled": true,
+				"staticDelayMS": 32,
+				"leftEyeDelayMS": 7.5,
+				"rightEyeDelayMS": 0,
+				"localTimeOverride": true
+		},
+
         "timeWarp": {
             "enabled": true,
             "asynchronous": false,
```

Comment:
This and other config file changes that add prediction and change the overfill factor are all consistent with what I'm expecting to see in the repository.